README.md

```diff
@@ -1,43 +1,31 @@
 # Form validation problem
 
-We've created this problem to evaluate how developers tackle a real-world problem. If you've been assigned this problem you should spend around **2 hours** working on it. The last thing we want you to do is toil away for days on end!
+We've created this problem to evaluate how developers tackle a real-world problem. If you've been assigned this problem, you should spend around **3 hours** working on it. The last thing we want you to do is toil away for days on end!
 
 If you've stumbled across this and want to work at [Springload](https://www.springload.co.nz/) feel free to submit it too. We're always on the lookout for skilled developers.
 
 ## Problem definition
 
-Imagine, if you will, a form.
+We'd like you to create a web page with a form on it. The form should have these fields:
 
-The form has a title reading "Contact form".
+- `Email` must be a valid email address.
+- `Password` must be longer than 8 characters.
+- `Colour` is multi-choice, where only one option may be selected, and the options are `Blue`, `Green`, `Red`, `Black` and `Brown`.
+- `Animals` is multi-choice, where multiple options can be selected, and the options are `Bear`, `Tiger`, `Snake`, and `Donkey`.
+- If `Tiger` is selected then a textbox `Type of tiger` is revealed. This is a required field.
 
-The form has these fields:
+## Technical requirements
 
-* `Email` must be a valid email address.
-* `Password` must be longer than 8 characters.
-* `Colour` is multichoice, where only one option may be selected, and the options are `Blue`, `Green`, `Red`, `Black` and `Brown`.
-* `Animals` is multichoice, where multiple options can be selected, and the options are `Bear`, `Tiger`, `Snake`, and `Donkey`.
-* If `Tiger` is selected then a textbox `Type of tiger` is revealed and this is a required field.
-
-Feel free to decide any styling, responsive behaviour, or any text for error messages or labels or placeholders etc, yourself.
-
-Your objective is to produce this form with as much or as little sophistication as you see fit, explaining what informed your decisions.
-
-## Other advice
-
-2 hours isn't much, so it's ok to not do everything!
-
-We want to see what you've prioritised, and a description of what hasn't been done.
-
-You might want to consider:
-
-- Accessibility
-- Progressive enhancement
-- Browser support
-- Testing
-- Documentation
```

Comment:
@liamjohnston is there a particular reason why you've chosen to remove these hints here?

Reply:
I think they're either no longer relevant, or covered elsewhere in the readme.

- **accessibility** I specifically added as a requirement
- **progressive enhancement** I guess would be interesting to see if they add any and want to discuss it, but didn't feel it was really a requirement for this particular form scenario.
- **browser support** is less of a thing plaguing FEDs nowadays
- **testing** again probably not necessary, but anyone who feels it is could add them or discuss it as something they didn't do but might normally.
- **documentation** – we already ask for notes ("explain any compromises"), otherwise I'd like to see the code as not really needing documentation. I dunno, just one less thing to ask for I guess.

Thoughts?

Reply:
Hmm... Well, it could be a good thing to see if they even think about these things. OK.

Mind you, I'd expect them to add tests and docs of some sort.